examples/container.rs

```diff
@@ -18,78 +18,56 @@ fn main() -> Result<(), Infallible> {
     let mut display: SimulatorDisplay<Rgb888> = SimulatorDisplay::new(display_size);
 
     let character_style = MonoTextStyle::new(&FONT_10X20, Rgb888::BLACK);
-    let text1 = Text::new(
-        TextOptions {
-            character_style,
-            layout_options: Default::default(),
-        },
-        "Line 1",
-    );
-    let text2 = Text::new(
-        TextOptions {
-            character_style,
-            layout_options: Default::default(),
-        },
-        "Line 2",
-    );
+    let text1 = Text::new("Line 1", character_style);
+    let text2 = Text::new("Line 2", character_style);
 
-    let rgb_swatch = Container::new(Options {
-        alignment: Alignment::Center,
-        axis: Axis::Horizontal,
-        border: Some(Border {
+    let rgb_swatch = Container::new()
+        .alignment(Alignment::Center)
+        .axis(Axis::Horizontal)
+        .border(Some(Border {
             color: Rgb888::BLACK,
             width: 1,
-        }),
-        children: vec![
+        }))
+        .corner_radii(Some(CornerRadii::new(Size::new(10, 10))))
+        .justification(Justification::SpaceBetween)
+        .padding(Insets::all(10))
+        .width(Some(200))
+        .children(vec![
             colored_container(Rgb888::RED, 25).boxed(),
             colored_container(Rgb888::GREEN, 40).boxed(),
             colored_container(Rgb888::BLUE, 55).boxed(),
```

Comment:
Chaining like these seems much nicer than the `Options` structs I was using before.